scripts/verifyInstallPage.js

```diff
@@ -0,0 +1,140 @@
+/* eslint-disable no-console */
+const fs = require('fs');
+const yaml = require('js-yaml');
+const path = require('path');
+
+const verifySteps = (steps, installPage) => {
+  let issues = 0;
+  steps.forEach((step) => {
+    if (!step.filePath.includes(`/install/${installPage}/`)) {
+      console.error(
+        '\x1b[31m%s\x1b[0m',
+        `\n (!) ${installPage}.yaml`,
+        `\x1b[33m${step.filePath}\x1b[0m must point to a file in \x1b[32msrc/install/${installPage}/\x1b[0m \n`
+      );
+      issues = ++issues;
+      step.overrides?.forEach((override) => {
+        if (!override?.filePath?.includes(`/install/${installPage}/`)) {
+          console.error(
+            '\x1b[31m%s\x1b[0m',
+            `\n (!) ${installPage}.yaml`,
+            `\x1b[33m${step.filePath}\x1b[0m must point to a file in \x1b[32msrc/install/${installPage}/\x1b[0m \n`
+          );
+          issues = ++issues;
+        }
+      });
+    }
+  });
+  if (issues > 0) {
+    console.error(
+      '%s\x1b[31m%s\x1b[0m',
+      'please fix ',
+      issues,
+      'issue/s above \n'
```

Comment:
`huge nit` it cool if we change this to "issue(s)" just think it'd be easier to read maybe 🤷

Reply:
ooh yeah i'm down